admin/tests/integration/components/administration/organizations-batch-update_test.js

```diff
@@ -0,0 +1,127 @@
+import { render } from '@1024pix/ember-testing-library';
+import Service from '@ember/service';
+import { triggerEvent } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import ENV from 'pix-admin/config/environment';
+import { module, test } from 'qunit';
+import sinon from 'sinon';
+
+import setupIntlRenderingTest from '../../../helpers/setup-intl-rendering';
+
+const accessToken = 'An access token';
+const fileContent = 'foo';
+const file = new Blob([fileContent], { type: `valid-file` });
+
+module('Integration | Component |  administration/organizations-batch-update', function (hooks) {
+  setupIntlRenderingTest(hooks);
+
+  let fetchStub;
+
+  hooks.beforeEach(function () {
+    class SessionService extends Service {
+      data = { authenticated: { access_token: accessToken } };
+    }
+    this.owner.register('service:session', SessionService);
+
+    fetchStub = sinon.stub(window, 'fetch');
+  });
+
+  hooks.afterEach(function () {
+    window.fetch.restore();
+  });
+
+  module('when import succeeds', function (hooks) {
+    hooks.beforeEach(function () {
+      fetchStub
+        .withArgs(`${ENV.APP.API_HOST}/api/admin/organizations/add-organization-features`, {
+          headers: {
+            Authorization: `Bearer ${accessToken}`,
+            'Content-Type': 'text/csv',
+            Accept: 'application/json',
+          },
+          method: 'POST',
+          body: fileContent,
+        })
+        .resolves(fetchResponse({ status: 204 }));
+    });
+
+    test('it displays a success notification', async function (assert) {
+      // given
+      const notificationSuccessStub = sinon.stub();
+      class NotificationsStub extends Service {
+        success = notificationSuccessStub;
+        clearAll = sinon.stub();
+      }
+      this.owner.register('service:notifications', NotificationsStub);
+
+      // when
+      const screen = await render(hbs`<Administration::OrganizationsBatchUpdate />`);
+      const input = await screen.getByLabelText(
+        this.intl.t('components.administration.organizations-batch-update.upload-button'),
+      );
+      await triggerEvent(input, 'change', { files: [file] });
+
+      // then
+      assert.ok(true);
+      sinon.assert.calledWith(
```

Comment:
**suggestion :**

`assert.ok(notificationSuccessStub.calledWith(this.intl.t('components.administration.organizations-batch-update.notifications.success')));`